mappings/net/minecraft/server/command/ReloadCommand.mapping

```diff
@@ -2,3 +2,16 @@ CLASS net/minecraft/class_3097 net/minecraft/server/command/ReloadCommand
 	FIELD field_25343 LOGGER Lorg/apache/logging/log4j/Logger;
 	METHOD method_13529 register (Lcom/mojang/brigadier/CommandDispatcher;)V
 		ARG 0 dispatcher
+	METHOD method_13530 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD method_13531 (Lnet/minecraft/class_2168;)Z
+		ARG 0 source
+	METHOD method_29478 getNewDatapacks (Lnet/minecraft/class_3283;Lnet/minecraft/class_5219;Ljava/util/Collection;)Ljava/util/Collection;
+		ARG 0 resourcePackManager
+		ARG 1 saveProperties
+		ARG 2 enabledDatapacks
```

Comment:
```suggestion
		ARG 2 enabledDataPacks
```
Just fyi, this is how we capitalize "data pack". It's two words, like "item stack".